1.6/Source/MSSFP/Questing/ScenPart_Pursuers.cs

```diff
@@ -81,6 +97,40 @@ public class ScenPart_Pursuers : ScenPart
     public List<int> tmpRaidValues;
     public int startMapId = -1;
 
+    public int initialWarningDelay = 2700;
+    public int initialRaidDelay = 30000;
+    public IntRange warningDelayRange = new(840000, 960000);
+    public IntRange raidDelayRange = new(1080000, 2100000);
+    public int minRaidPoints = 5000;
+    public float raidPointMultiplier = 1.5f;
+    public int gravEngineCheckInterval = 2500;
+    public FactionDef faction;
+    public PawnsArrivalModeDef raidArrivalMode;
+    public List<MapGeneratorDef> safeMapGenerators = [];
+    public List<LandmarkDef> safeLandmarks = [];
+    public string alertPursuerThreatCriticalText = "Incoming Pursuers";
+    public string alertPursuerThreatText = "Pursuers";
+    public string alertPursuerThreatCriticalDescText = "They've found you, they will not stop";
+    public string alertPursuerThreatDescText = "The pursuers are closing in on your location";
+    public string letterLabelPursuerThreat = "Incoming Pursuers";
+    public string letterTextPursuerThreat =
+        "Pursuers are close to determining your location. When they locate you, they will send a massive force to kill you.\\n\\nGather your supplies, then flee!";
```

Comment:
I intentionally didn't put the strings here so they have to be set in XML so that they can be translated properly with definjecteds. Should throw exceptions in config errors instead of defaulting.

Reply:
I made them translation strings instead so we can pass in the faction name for the messages